### PR TITLE
feat : added homepage

### PR DESCRIPTION
**Pull request by someshtiwarii**
Pages:
Homepage
Products(dummy page)
Cart(dummy page)
Signup(dummy page)
Login(dummy page)
Wishlist (dummy page)

Components:
Navbar
Categories

Assets:
Images
Favicon

### CODE REVIEW

File: src/Routes/NavRoute.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Routes, Route } from "react-router-dom";
+import { Home } from "../pages/Home/Home";
+import { Products } from "../pages/Products/Products";
+import { Cart } from "../pages/Cart/Cart";
+import { Signup } from "../pages/Signup/Signup";
+import { Login } from "../pages/Login/Login";
+import { Wishlist } from "../pages/Wishlist/Wishlist";
```
Comment (gopalbbharadva) [inline src/Routes/NavRoute.jsx:7]: Suggestion : You can make a barrel export for pages.

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import { Navbar } from "../../components/Navbar/Navbar";
+import { Category } from "../../components/Category/Category";
+
+const Home=()=>{
+
+const [categories, setCategories] = useState([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/categories");
```
Comment (gopalbbharadva) [inline src/pages/Home/Home.jsx:14]: Suggestion: You can make services for each api call.

**Review by gopalbbharadva — APPROVED** [review]
UI looks clean. Good to go

**Review by manojkumar8400 — APPROVED** [review]
UI Looks great.good to go.

**Review by bipin7yadav — APPROVED** [review]
ui looks cool

File: src/pages/Home/Home.jsx (hunk 2 above)
Comment (someshtiwarii) [reply src/pages/Home/Home.jsx:14]: i will in my upcoming PRs

**Pull request merged**